librz/asm/arch/rl78/rl78.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: 2023 Bastian Engel <bastian.engel00@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RL78_H
+#define RL78_H
+
+#include "instr.h"
+
+#include <rz_util.h>
+#include <rz_types.h>
+
+#include <stddef.h>
+#include <stdbool.h>
+
+/**
+ * \brief Disassemble a byte sequence into an rl78_instr
+ * \param instr Resulting instruction
+ * \param byte_read Number of bytes disassembled
+ * \param buf Byte buffer
+ * \param buf_len Length of the buffer
+ * \return false if byte sequence does not represent a valid instruction
```

Comment:
`if` -> `If`.

Reply:
By the way, we prefer using Doxygen documentation in the function definition place (wherever body is)